### DISCUSSION TITLE
Disable  control - c  and right click copy and paste  on site

### DISCUSSION THREAD

**Discussion by IoTSIGithub** [chamilo/chamilo-lms#5318] [General]
Hello
Is there a means by which to disable copying and pasting of content site wide?
We would like students not to be able to copy any of the courseware . It protects our Ip and stops unwarted circulation.
Any assistance greatly appreciated.

**Comment by Kaneda-1** [comment]
What you request is not 100% possible,. Check the configuration.php file options.

Example: [Check the configuration.php file options in](https://github.com/chamilo/chamilo-lms/blob/1.11.x/main/install/configuration.dist.php)

>  Block copy/paste/save/print keys and right-clicks in exercises
//$_configuration['quiz_prevent_copy_paste'] = false;

**Reply by IoTSIGithub** [reply]
Thank you for the reply. Just wondering what folder is this file located in?

**Reply by contidos** [reply]
That file is located in the following route: /app/config  and in there you will find the file configuration.php
